socorro/signature/siglists/irrelevant_signature_re.txt

```diff
@@ -162,6 +162,8 @@ mozilla::ThreadSafeAutoRefCnt
 mozilla::UniquePtr
 mozilla::widget::WlCrashHandler
 mozilla::wr::RenderThread::AddRef
+~nsCOMPtr
+~nsCOMPtr_base
```

Comment:
These show up in the signature like `nsCOMPtr_base::~nsCOMPtr_base`, so these two lines have no effect on signature generation.

Further, `nsCOMPtr` is in the prefix list so signature generation already covers those frames.

Reply:
Ah, yeah, I somehow managed to realize that the regular expression had to match from the start of the line. I'll update these.

Reply:
I've update this in a new patch. I don't know if there's a good way to rebase the two patches together from the GitHub UI or not.

Reply:
I can rebase it.

Reply:
Thanks